Signal/src/util/OWSBackupJob.h

```diff
@@ -11,10 +11,42 @@ extern NSString *const kOWSBackup_ManifestKey_EncryptionKey;
 extern NSString *const kOWSBackup_ManifestKey_RelativeFilePath;
 extern NSString *const kOWSBackup_ManifestKey_DataSize;
 
+@class OWSBackupIO;
+@class OWSBackupJob;
+@class OWSBackupManifestContents;
+
 typedef void (^OWSBackupJobBoolCompletion)(BOOL success);
 typedef void (^OWSBackupJobCompletion)(NSError *_Nullable error);
+typedef void (^OWSBackupJobManifestSuccess)(OWSBackupManifestContents *manifest);
+typedef void (^OWSBackupJobManifestFailure)(NSError *error);
 
-@class OWSBackupJob;
+@interface OWSBackupManifestItem : NSObject
+
+@property (nonatomic) NSString *recordName;
+
+@property (nonatomic) NSData *encryptionKey;
+
+// This property is only set for certain types of manifest item;
+@property (nonatomic, nullable) NSString *relativeFilePath;
```

Comment:
Can you expound on the comment as to which types?

Reply:
Done.